### DISCUSSION TITLE
Where are the Yimmenu files stored

### DISCUSSION THREAD

**Discussion by MeekGroundhog21** [YimMenu/YimMenu#2798] [Q&A]
So i had to reinstall Windows and lost all the settings saved so im wondering where are the files stored so next time i could save it on my hard drive the next time i need to reinstall windows

**Comment by BumFellow** [comment]
C:\Users\MeekGroundhog21\AppData\Roaming\YimMenu\

**Reply by MeekGroundhog21** [reply]
Thanks

**Comment by xesdoog** [comment]
_%AppData%\yimmenu_
Copy/Paste that in your file explorer address bar or press Start +R, paste it then hit Enter.
